lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -1675,6 +1675,33 @@ bool ProxyShape::primHasExcludedParent(UsdPrim prim)
   return false;
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+MString ProxyShape::recordUsdPrimToMayaPath(const UsdPrim &usdPrim,
+                                            const MObject &mayaObject){
+  // Retrieve the proxy shapes transform path which will be used in the
+  // UsdPrim->MayaNode mapping in the case where there is delayed node creation.
+  MFnDagNode shapeFn(thisMObject());
+  const MObject shapeParent = shapeFn.parent(0);
+  MDagPath mayaPath;
+  MDagPath::getAPathTo(shapeParent, mayaPath);
```

Comment:
This feels a little flimsy. It could occur that this object has multiple paths, is there a way that we could isolate a required path? (Ok, I'm assuming the answer is probably no - but just raising it as a consideration, which at the very least should be added to the known issues?)